INV4/pallet-inv4/src/ips.rs

```diff
@@ -613,12 +658,14 @@ impl<T: Config> Pallet<T> {
                 default_permission: ipl_default_permission,
             };
 
+            // ???
```

Comment:
This is minting existential deposit of IPT0 to the creator of the replica, don't worry about commenting this as it's obsolete and needs to be rewritten.